.github/workflows/linux_ci.yml

```diff
@@ -0,0 +1,36 @@
+on: [push]
+
+name: Linux CI
+
+jobs:
+  check:
+    name: Compile and Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install gtk3 and dependencies.
+        run: sudo apt-get update && sudo apt install menu libcurl4-openssl-dev libelf-dev libdw-dev binutils-dev libgtk-3-dev libmount-dev libsoup2.4-dev libxtst-dev at-spi2-core openbox
```

Comment:
Same as above.

Reply:
These dependencies in here reflect what was being removed in the Travis CI. That is, these are being kept in an attempt to get the OpenBox approach to work again. However, I'm having some trouble with that when I was trying to get #48 working. See [here.](https://github.com/divark/gtk-test/runs/4400876756?check_suite_focus=true)

Reply:
Nevermind, I got this to work using [xvfb-action](https://github.com/GabrielBB/xvfb-action). Testing locally using act, it looks like we still need `libxdo-dev` to get the tests to run without a linker error.